src/messages/messagesEN.ts

```diff
@@ -66,7 +66,14 @@ const messages = {
       ingressUnauthenticated:
         "Do you want to use AI in your teaching? NDLA has created two chatbots that protect your privacy and can be safely used for work and in teaching. During exams the county may deny access to the chatbots. Log in to to access the chat bots.",
     },
-    resource: { addedToFolder: 'Resource added to "{{folder}}"', added: "Added", removed: "Removed" },
+    resource: {
+      addedToFolder: 'Resource added to "{{folder}}"',
+      added: "Added",
+      removed: "Removed",
+      showTags: "Show tags",
+      tagsDialogTitle: "Tags related to resource {{title}}",
+      noTags: "No tags ...",
```

Comment:
Er dette en trussel? La oss bare ha "No tags.". Kan eventuelt droppe å vise valget dersom det ikke er tags på ressurser, eller er det mye stress?

Reply:
Ja, det var tenkt som en trussel .. 
Hmm, det er ikke spesielt stress, men jeg er ikke veldig fan av å skjule/vise sånne valg basert på innholdet egentlig, er jo ikke selvforklarende da hvorfor det vises på noen og ikke på andre? Men lov å være uenig her 😅 Jeg tenkte på å disable den, men så ble jeg usikker på hvordan vi skulle gjort det uu-vennlig eventuelt 😅

Reply:
Usikker på om det kan gjøres UU-vennlig. Det burde gå fint å disable i Menu, men det går nok ikke like fint i toolbar-modalen.

Reply:
Jeg stemmer for å beholde det sånn det er nå!